src/components/MatchedRoutes/MatchedRoutes.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { FC } from 'react';
+
+
+interface RouteData {
+  routeId: number;
+  name: string;
+  distanceFromOrigin: string;
+  distanceFromDestination: string;
+  time: string;
+  days: string[];
+}
+
+interface MatchedProps {
+  routes: Array<RouteData>
+}
+
+
+const MatchedRoutes: FC<MatchedProps> = ({ routes }) => {
+
+  const formatTime = (time: string): string => {
+    let hour: string | any = time.split(':').shift();
+    if (parseInt(hour) >= 12) {
+      return 'pm'
+    } else {
+      return 'am'
+    }
+  }
```

Comment:
Maybe there is a way to use this functionality on the detailed routes component as well so we don't have to redo it there